.github/workflows/benchmarks.yml

```diff
@@ -66,6 +66,11 @@ jobs:
           path: benchmarks/data/kurucz_cd23_chianti_H_He.h5
           key: atom-data
 
+      - name: Copy atom data to refdata
+        run: |
+          mkdir -p tardis-refdata/atom_data
+          cp benchmarks/data/kurucz_cd23_chianti_H_He.h5 tardis-refdata/atom_data/
```

Comment:
I will suggest using the setup lfs action-
https://github.com/tardis-sn/tardis/blob/e88a31a809eb041068fffeeb16e1585b8e51f3a5/.github/workflows/tests.yml#L57-L58


Right now, there is a step in this workflow which uses Azure to download the atomic file. We're moving away from Azure. Since we already have caches, I'd suggest reusing the cache here and copying the file over to the benchmarks folder. That way, we won't have extra cache(we're near the limit).